pom.xml

```diff
@@ -312,6 +312,16 @@
     </dependencyManagement>
 
     <dependencies>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>${slf4j.version}</version>
```

Comment:
✏️ These 2 dependencies should be duplicated in the `dependencyManagement` section along with the `<version>` and in here, just leave them but remove the `<version>`